script.embuary.helper/resources/lib/kodi_monitor.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python
+
+########################
+
+import xbmc
+import xbmcgui
+import time
+import json
+
+from resources.lib.helper import *
+
+########################
+
+class KodiMonitor(xbmc.Monitor):
+
+    def __init__(self):
+        xbmc.Monitor.__init__(self)
```

Comment:
not needed

Reply:
Why is it not needed? 
It's part of the service.